README.md

```diff
@@ -18,6 +18,33 @@ This action is used by the Dependabot [version][docs-version-updates] and [secur
 To manually upgrade `dependabot-action` on your [GitHub Enterprise Server (GHES)](https://github.com/enterprise), follow [these instructions](https://docs.github.com/en/enterprise-server/admin/managing-github-actions-for-your-enterprise/managing-access-to-actions-from-githubcom/manually-syncing-actions-from-githubcom).
 **Warning:** The current release of `dependabot-action` only guarantees backwards compatibility with the [currently supported GHES versions](https://docs.github.com/en/enterprise-server/admin/all-releases). Once a GHES version is deprecated, future versions of `dependabot-action` may introduce incompatible changes.
 
+##  Steps to Update Your PR and Deploy Changes
+1. Make changes to the PR
+2. Run the following command in your terminal on the same branch used to create the PR whenever you make changes to the PR
```

Comment:
I think we can be a little more terse by assuming anyone reading this is familiar with editing/testing a PR requires being on the same branch... 

```suggestion
2. Run the following commands:
```